docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,342 @@
+
+Once the installation is complete, you can test the Serverless CLI by displaying the commands available to you:
+
+```shell
+npx serverless help
+```
+
+You're now set up with the Serverless tools! Let's move on to preparing your Bolt app to run as an AWS Lambda function.
+
+---
+
+### Get a Bolt Slack app
+
+If you haven't already built your own Bolt app, you can use our [Getting Started guide][getting-started-guide] or clone the template app below:
+
+```shell
+git clone https://github.com/slackapi/bolt-js-getting-started-app.git
+```
+
+After you have a Bolt app, navigate to its directory:
+
+```shell
+cd bolt-js-getting-started-app/
+```
+
+Now that you have an app, let's prepare it for AWS Lambda and the Serverless Framework.
+
+---
+
+### Prepare the app
+
+**1. Prepare the app for AWS Lambda**
+
+In this section, we'll update your Bolt app to respond to a Lambda function instead of listening for HTTP requests. We'll do this by creating a custom Bolt receiver for Lambda.
+
+First, install the official [AWS Serverless Express](https://github.com/awslabs/aws-serverless-express) module to transform Express HTTP requests to Lambda function events:
+
+```bash
+npm install aws-serverless-express
+```
+
+Next, update the [source code that imports your modules](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L1) in `app.js` to require Bolt's Express receiver and the AWS Serverless Express module:
+
+```javascript
+const { App, ExpressReceiver } = require('@slack/bolt');
+const awsServerlessExpress = require('aws-serverless-express');
+```
+
+Then update the [source code that initializes your Bolt app](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L3-L7) to create a custom receiver using AWS Serverless Express:
+
+```javascript
+// Initialize your custom receiver and ack Slack requests after handled
+const expressReceiver = new ExpressReceiver({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  processBeforeResponse: true
+});
+
+// Initializes your app with your bot token and signing secret
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  receiver: expressReceiver
+});
+
+// Initialize your AWSServerlessExpress server using Bolt's ExpressReceiver
+const server = awsServerlessExpress.createServer(expressReceiver.app);
+```
+
+Finally, at the bottom of your app, update the [source code that starts the HTTP server](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L40-L45) to now respond to an AWS Lambda function event:
+
+```javascript
+// Handle the Lambda function event
+module.exports.handler = (event, context) => {
+  console.log('⚡️ Bolt app is running!');
+  awsServerlessExpress.proxy(server, event, context);
+};
+```
+
+When you're done, your app should look similar to the ⚡️[Getting Started with Serverless Framework app][getting-started-with-serverless-framework-app/app.js].
```

Comment:
This link will work after the PR is merged, until then you can view the `app.js` at:
https://github.com/mwbrooks/bolt-js/blob/220-faas-deployment-guides/examples/getting-started-with-serverless-framework/app.js